### PR TITLE
Add missing test from keylime testsuite to e2e plan

### PR DESCRIPTION
**Pull request by Koncpa**
Also set boolean faked_measured_boot_log to yes.

### CODE REVIEW

File: packit-ci.fmf [hunk 1]
```diff
@@ -40,12 +41,14 @@
...
      - /functional/db-mariadb-sanity-on-localhost
      - /functional/db-mysql-sanity-on-localhost
+     - /functional/durable-attestion-sanity-on-localhost
      - /functional/ek-cert-use-ek_check_script
      - /functional/ek-cert-use-ek_handle-custom-ca_certs
      - /functional/install-rpm-with-ima-signature
      - /functional/keylime-non-default-ports
      - /functional/keylime_tenant-commands-on-localhost
      - /functional/keylime_tenant-ima-signature-sanity
+     - /functional/measured-boot-policy-sanity
```
Comment (kkaarreell) [inline packit-ci.fmf:51]: @Koncpa This one should not be added

**Comment by kkaarreell** [comment]
> The new test (`durable-attestion-sanity-on-localhost`) is failing...

Since we have switched to the rust agent, in order to make these measured boot related tests to work we need to point them to the fake measured boot log through
```
environment:
  TPM_BINARY_MEASUREMENTS: /var/tmp/binary_bios_measurements
```
in each plan. 
Unfortunately, @Koncpa is away this week so this will have to wait a bit.

**Review by kkaarreell — CHANGES_REQUESTED** [review]

**Comment by maugustosilva** [comment]
@kkaarreell I can confirm that all tests pass with the changes you suggested (just FYI, @Koncpa)

File: packit-ci.fmf [hunk 2]
```diff
@@ -8,6 +8,7 @@
   context:
     swtpm: yes
     agent: rust
+    faked_measured_boot_log: yes
```
[the remark this answers is not in the document]
Comment (Koncpa) [reply packit-ci.fmf:12]: Shouled be resolved.

File: packit-ci.fmf (hunk 1 above)
Comment (Koncpa) [reply packit-ci.fmf]: DONE.

**Comment by maugustosilva** [comment]
@Koncpa I am seeing some really weird errors here, such as `line 1379: keylime_sign_runtime_policy: command not found`. Before we debug further, please rebase and let's rerun the tests.

**Pull request merged**